gulp-tasks/build-packages.js

```diff
@@ -25,39 +30,63 @@ const ERROR_NO_NAMSPACE = oneLine`
   JavaScript. Please fix for:
 `;
 
-/*
- * This function takes an object like { nested: { foo: bar } } and a string
- * like `nested.foo` and returns true if that value is defined on the obejct.
- */
-const testObjectValueExists = (object, nestedPath) => {
-  const pieces = nestedPath.split('.');
-  let currentRoot = object;
-  for (const piece of pieces) {
-    if (!currentRoot[piece]) {
-      return false;
-    }
-    currentRoot = currentRoot[piece];
+// This makes Rollup assume workbox-* will be added to the global
+// scope and replace references with the core namespace
+const globals = (moduleId) => {
+  // This regex matches for (workbox-*)(/any/path/here.mjs)
```

Comment:
Is there a need to use `path.sep` here to support Windows?

Also, could you get away with using `moduleId.split(path.sep)` instead of a RegExp? I love RegExps, but for this use case, they seem more fragile.